test/suites/test_schema.py

```diff
@@ -97,13 +135,7 @@ def verify_unicode_index(self, index):
         self.assertEqual(index.name, self.unicode_index_name_u)
         self.assertEqual(len(index.parts), 1)
 
-    def test_00_authenticate(self):
-        self.assertIsNone(self.srv.admin("box.schema.user.create('test', { password = 'test' })"))
-        self.assertIsNone(self.srv.admin("box.schema.user.grant('test', 'read,write', 'space', '_space')"))
-        self.assertIsNone(self.srv.admin("box.schema.user.grant('test', 'read,write', 'space', '_index')"))
-        self.assertEqual(self.con.authenticate('test', 'test')._data, None)
-
-    def test_01_space_bad(self):
+    def test_00_space_bad(self):
```

Comment:
You can just leave it here starting from 01

Reply:
Done